data/cube/CubeField.js

```diff
@@ -90,15 +96,19 @@ export class CubeField extends Field {
     parseAggregator(val) {
         if (isString(val)) {
             switch (val) {
+                case 'AVG':             return CubeField.averageAggregator;
+                case 'AVG_STRICT':      return CubeField.averageStrictAggregator;
+                case 'COUNT':           return CubeField.childCountAggregator; // TODO: Thought here is this is the more standard idea of "COUNT"
```

Comment:
Is the additional shorthand worth it or just confusing?

Reply:
I find it confusing.  Makes me think I have to use it, rather than learning what COUNT means